examples/cordova-sample/MyApp/www/js/index.js

```diff
@@ -82,6 +82,8 @@ const app = {
         }
     );
 
+    Purchases.invalidateCustomerInfoCache();
```

Comment:
Added as a way to keep api testing while coding this.

Reply:
maybe add that in as a comment?

Reply:
✅